src/components/timeline/Track.css

```diff
@@ -20,29 +23,29 @@
    * This variable will be used every time we need to calculate some width that
    * depends on this value, from the intended perceived width. */
   --selected-left-border-width: 3px;
+  --selected-left-border-color: var(--blue-60);
 
   display: flex;
   flex-flow: row nowrap;
-
-  /* This padding is added to the button's padding as the perceived padding. For
-   * a global row, it will be replaced with a margin when it's selected. */
-  padding-left: var(--selected-left-border-width);
   border-top: 1px solid var(--grey-30);
   background-color: #fff;
 }
 
+
+/* We use a colored pseudo element the track row instead of using border, so
+  * that when we select adjacent tracks, we see just one line, without any
+  * border on top of it. */
+.timelineTrackRow::before {
+  width: var(--selected-left-border-width);
+  content: "";
+}
+
+.timelineTrackRow.selected::before {
+  background-color: var(--selected-left-border-color);
+}
```

Comment:
Let me know if I should revert this and only tweak the color for HCM. I'm not sure I understand the original issue with border (except that then the top gray border doesn't seem to go all the way to the left ?)